app/ai/voice/agents/breeze_buddy/test/scenario_generator.py

```
@@ -0,0 +1,685 @@
+        scenario_id = d.get("id") or f"scenario_{uuid.uuid4().hex[:8]}"
+        return GeneratedScenario(
+            id=scenario_id,
+            name=d.get("name", scenario_id),
+            scenario_type=scenario_type,
+            description=d.get("description") or None,
+            payload_example=d.get("payload_example") or {},
+            turns=turns,
+            expected_outcome=d.get("expected_outcome") or None,
+            expected_final_node=d.get("expected_final_node") or None,
+        )
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)